apps/server/src/classes/simple-timer/SimpleTimer.ts

```diff
@@ -37,6 +37,12 @@ export class SimpleTimer {
     return this.state;
   }
 
+  public addTime(millis: number, timeNow: number): SimpleTimerState {
+    this.state.duration += millis;
+    this.state.current += millis; // optimistic update of curret value in case of pause or stop
+    return this.update(timeNow);
```

Comment:
i dont think these function should do multiple things
if what we want is to add time and update, then we should call the two things